### ISSUE TITLE
my doge core wallet was emptied, how did they get my private key?

### ISSUE THREAD

**Issue by mmoren73** [dogecoin/dogecoin#1504]
somehow a person got my priviate key and cleaned out my doge wallet, i had 13,697, is there anything i can do to get them back,  i have no idea how they did it, keylogger maybe, i dont store passwords or paraphrases on my pc, all written in a notebook, and no cells or webcams that can see them either. im at a loss, literally. is there anything i can do, thank god it wasnt much but it was all the doge i had amd was a lot of work

**Comment by mmoren73** [comment]
whoever got my doge coins ended up going to a wallet with like 7 million doge coins plus,

**Comment by rnicoll** [comment]
One of the key factors in a decentralised currency is that there's no central authority that can override things, sorry, we can't help.

The most common ways I've seen keys compromised is by uploading them unencrypted to Dropbox, using a third party to generate the keys (such as a novelty address generator, as it means they have the key before you do), or simply a virus on your PC as you suggested. Personally I use https://www.ledgerwallet.com/products/ledger-nano-s as a solution but they're expensive sadly.

**Comment by mmoren73** [comment]
hey micoll, update, i dont think i was hacked, but for some reason, all my coins went to a watch only address, so my wallet no says 67 spendable, and 18,761 watch only. how did that happen?, and how do i get them back, the watch only address is inmy receive keys, but i dont recognize it. but that what i should have for coins is the 18,761. is there a way to get my coins back to my spenable wallet. i know the very basics, of the console, but i dont know how to get controll of that watch only key to egt my coins back?

**Comment by Fulvio55** [comment]
If you haven't already, get a copy of ALL you keys out NOW! There's an ELI5 with the process on /r/dogecoin.

**Comment by D3m0nKingx** [comment]
@Fulvio55 I've reported him. Ignore any further comments, github admins will address this.
